File: 006/Problem6.cs
using System;

namespace EulerProblems
{
    class Program
    {
        static void Main(string[] args)
        {
            var watch = new System.Diagnostics.Stopwatch();

            watch.Start();
            Problem();
            watch.Stop();

            Console.WriteLine($"Function Took: {watch.Elapsed}");
        }

        private static int SumOfSquares(int num)
        {
            var sum = 0;
            while (num > 0)
            {
                sum += num * num;
                num--;
            }
            return sum;
        }

        private static int SquareOfSum(int num)
        {
            var sum = 0;
            while (num > 0)
            {
                sum += num;
                num--;
            }

            return sum * sum;
        }
        private static void Problem()
        {
            const int naturalNum = 100;
            Console.WriteLine(SquareOfSum(naturalNum) - SumOfSquares(naturalNum));
        }
    }
}

File: 006/Problem6.dart
void main() {
  var numLoops = 100000;
  final stopwatch = Stopwatch()..start();
  for (var i = 0; i < numLoops; i++) {
    problem();
  }
  final elapsedTime = stopwatch.elapsed;
  print(elapsedTime);
}

int sumOfSquares(n) {
  var sum = 0;
  for (var num = 1; num <= n; num++) {
    sum += num * num;
  }
  return sum;
}

int squareOfSum(n) {
  var sum = 0;
  for (var num = 1; num <= n; num++) {
    sum += num;
  }
  return sum * sum;
}

void problem() {
  var naturalNum = 100;
  print(squareOfSum(naturalNum) - sumOfSquares(naturalNum));
}